QuadPaySDK/Widget/Widget.swift

```diff
@@ -235,10 +256,27 @@ extension Widget{
     func calculateInstalment() -> String{
         formatter.maximumFractionDigits = 2
         formatter.minimumFractionDigits = 2
-        formatter.currencyCode="USD"
+        formatter.currencyCode = "USD"
         formatter.numberStyle = .currency
         
-        let amount  = Double(amount) ?? 0.00
+        var amount  = Double(amount) ?? 0.00
+        var maxTier: Double = 0
+    
+        
+        for(_,element) in feeTiers.enumerated() {
```

Comment:
We can shift this code to when we make the call to fetch widget data.  That way we would have the `maxFee` stored upfront.  Same with `hasFees`